src/main/java/io/lightplugins/vaulty/Vaulty.java

```diff
@@ -5,6 +5,8 @@
 
 public class Vaulty extends JavaPlugin {
 
+    // The main class
+
```

Comment:
This is an example of an in-line comment.

Reply:
thats true